src/geocat/viz/util.py

```diff
@@ -1240,19 +1431,19 @@ def set_vector_density(data: xr.DataArray,
         return ds
 
 
-def get_skewt_vars(p: Quantity, tc: Quantity, tdc: Quantity,
-                   pro: Quantity) -> str:
+def get_skewt_vars(pressure: Quantity, temperature: Quantity,
+                   dewpoint: Quantity, pro: Quantity) -> str:
```

Comment:
should we change this to `profile` as well while we're breaking things?  I'm not particularly attached, but do think it's more clear/helpful to the user.

Reply:
Oh good point. We should support both kwargs for now with a warning. What do you want to change to `profile`?

Reply:
ah, sorry.  that was super unclear.

`pro` to `profile`

maybe a personal bias, but `pro` is pretty vague and `profile` at least gets me thinking in the right direction.

Reply:
Oh I should have seen that and made that connection honestly. My eyes didn't even see the `pro` kwarg